src/components/NextStep/Wrapper.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  bottom: 0px;
```

Comment:
Remove unnecessary units when use `0`

Reply:
I agree :)

Reply:
we can add a prettier option for yes. makes life easier.